crawlers/mooncrawl/mooncrawl/moonworm_crawler/crawler.py

```diff
@@ -172,6 +178,10 @@ def get_crawl_job_entries(
     """
     query = f"#status:active #type:{crawler_type} #subscription_type:{subscription_type.value}"
 
+    if extend_tags is not None:
+        for tag in extend_tags:
+            query += f" #{tag}"
```

Comment:
Better to add space cleaning to avoid incorrect results `query += f" #{tag.replace(' ', '')}"`